lib/api.ts

```diff
@@ -64,6 +64,8 @@ function makeSharedAPI(
   const { user, parameters, locales, ids, initialContentTypes, hostnames, release } = data
   const currentLocation = data.location || locations.LOCATION_ENTRY_FIELD
 
+  const effectiveIds = release ? { ...ids, release: release.sys.id } : ids
```

Comment:
I'm not against this approach per se -- but I wonder why we append the `releaseId` here in the SDK instead of doing so upstream ie in the channel sender that provides the `data` attribute (where the other `ids` are).